.github/workflows/rebuildDependencies.yml

```diff
@@ -17,37 +17,44 @@ jobs:
             os: macos-13
             preset: macos-conan-ninja-release
             conan_profile: macos-intel
+            conan_prebuilts: dependencies-mac-intel
             conan_options: --options with_apple_system_libs=True
             artifact_platform: intel
           - platform: mac-arm
             os: macos-13
             preset: macos-arm-conan-ninja-release
             conan_profile: macos-arm
+            conan_prebuilts: dependencies-mac-arm
             conan_options: --options with_apple_system_libs=True
             artifact_platform: arm
           - platform: ios
             os: macos-13
             preset: ios-release-conan-ccache
             conan_profile: ios-arm64
+            conan_prebuilts: dependencies-ios
             conan_options: --options with_apple_system_libs=True
           - platform: mingw
-            os: ubuntu-22.04
+            os: ubuntu-24.04
             preset: windows-mingw-conan-linux
             conan_profile: mingw64-linux.jinja
+            conan_prebuilts: dependencies-mingw
```

Comment:
let's call it mingw-64 perhaps?

Reply:
Sure. Might be better to rename profile itself to mingw-64 as well